modules/reactor/include/meshgenerators/CoreMeshGenerator.h

```diff
@@ -0,0 +1,66 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "MeshGenerator.h"
+
+/**
+ * Generates a pin-like structure for a square or hexagonal grid with the option to be 2- or 3-D.
```

Comment:
same

Reply:
2- => 2-D?

Reply:
Nah. It s just not describing the core mesh generator very well.
Seems just to be talking about a grid